### DISCUSSION TITLE
Rust book ??

### DISCUSSION THREAD

**Discussion by az4aaz** [Fabric-Development/fabric#18] [Q&A] [answered]
What about you use a doc generation / presentation system more specific for the language you use (python) such as Sphynx and Read the docs? It's a bit uncommon to see a python-driven project using the classic Rust documentation thing.... But nice project bros :+1:

**Comment by its-darsh** [comment, marked as answer]
what do you mean by "Rust documentation thing", if you mean mdbook, mdbook is a generic tool that works for everything, it doesn't mean that it's a "Rust ... thing" if it was developed by/written in rust, and mdbook was a nice choice for me since it's simple and straightforward, it's worth noting that Sphinx is not a markdown renderer but it's a set of tools to generate documentations out from python code

**Reply by az4aaz** [reply]
You are right I was drunk
